data/tlg0062/tlg001/tlg0062.tlg001.perseus-grc2.xml

```diff
@@ -1,22 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
  <teiHeader xml:lang="eng">
 <fileDesc>
 <titleStmt>
 <title xml:lang="grc">Φάλαρις</title>
 <author>Lucian of Samosata</author>
-<editor>A. M. Harmon</editor>
+<editor>Austin Morris Harmon, 1878-1950</editor>
 <sponsor>Perseus Project, Tufts University</sponsor>
 <principal>Gregory Crane</principal>
 <respStmt>
 <resp>Prepared under the supervision of</resp>
 <name>Bridget Almas</name>
 <name>Lisa Cerrato</name>
+<name>Gregory Crane</name>
```

Comment:
your name is already here

Reply:
My name is everywhere as Principal in a ceremonial way. I need different when I toiled over the file myself.